src/Castle.Windsor/Windsor/Installer/FromAssembly.cs

```diff
@@ -115,13 +115,21 @@ public static IWindsorInstaller InDirectory(AssemblyFilter filter, InstallerFact
 		///     <c>MyApp.SuperFeatures.dll</c>, <c>mscorlib.dll</c> and <c>ThirdPartyCompany.UberControls.dll</c> the <c>MyApp.exe</c> and <c>MyApp.SuperFeatures.dll</c> will be scanned for installers, and other
 		///     assemblies will be ignored.
 		/// </remarks>
+#if FEATURE_GETCALLINGASSEMBLY
 		[MethodImpl(MethodImplOptions.NoInlining)]
 		public static IWindsorInstaller InThisApplication()
 		{
 			var assembly = Assembly.GetCallingAssembly();
 			return ApplicationAssemblies(assembly, new InstallerFactory());
 		}
+#else
+		public static IWindsorInstaller InThisApplication(Assembly callingAssembly)
```

Comment:
I asked below for the new argument to be named `assembly`, I think this one (`InThisApplication`) should be named `rootAssembly` since that is consistent with `ApplicationAssemblies` and calls out that it can't just be any assembly because it'll recursively follow referenced assemblies.